packages/core/src/TestComponents.stories.tsx

```diff
@@ -0,0 +1,458 @@
+import { Variants as TagsInputVariantsStory } from "./TagsInput/TagsInput.stories";
+import { Variants as TextAreaVariantsStory } from "./TextArea/TextArea.stories";
+import { Multiple as ToggleButtonMultipleStory } from "./ToggleButton/ToggleButton.stories";
+import { Test as TypographyTestStory } from "./Typography/Typography.stories";
+import { setupChromatic } from ".storybook/setupChromatic";
+
+const meta: Meta = {
+  title: "Components/Test",
+};
+export default meta;
+
+export const Test: StoryObj = {
+  parameters: {
+    ...setupChromatic(
+      [
+        "DS3 dawn",
+        "DS3 wicked",
+        "DS5 dawn",
+        "DS5 wicked",
+        "Pentaho+ dawn",
+        "Pentaho+ wicked",
+      ],
+      5000,
+    ),
+    docs: { disable: true },
+    a11y: {
+      disable: true,
+    },
+  },
+  tags: ["skipTestRunner"],
+  play: async ({ canvasElement }) => {
+    // Accordion disabled story
+    const canvas = within(canvasElement);
+    const button = canvas.getByRole("button", { name: "System" });
+    await userEvent.click(button);
+  },
+  render: (args, context: any) => (
+    <div style={{ display: "flex", gap: 5 }}>
+      <div>
+        <HvSimpleGrid
+          cols={3}
+          style={{ alignItems: "start", justifyContent: "start" }}
+        >
+          <div>
+            <div style={{ display: "flex", height: 180 }}>
+              {AccordionDisabledStory.render?.(
+                AccordionDisabledStory.args as any,
+                context,
+              )}
+              {DotPaginationMainStory.render?.(
+                DotPaginationMainStory.args as any,
+                context,
+              )}
+              {DropDownMenuMainStory.render?.(
+                DropDownMenuMainStory.args as any,
+                context,
+              )}
+            </div>
+            <div>
+              {EmptyStateMinimalStory.render?.(
+                EmptyStateMinimalStory.args as any,
+                context,
+              )}
+              {EmptyStateWithActionStory.render?.(
+                EmptyStateWithActionStory.args as any,
+                context,
+              )}
+            </div>
+          </div>
+          {CheckBoxGroupVariantsStory.render?.(
+            CheckBoxGroupVariantsStory.args as any,
+            context,
+          )}
+          <div style={{ display: "flex", flexWrap: "wrap", gap: 5 }}>
+            {InputVariantsStory.render?.(
+              InputVariantsStory.args as any,
+              context,
+            )}
+          </div>
+        </HvSimpleGrid>
+        <HvSimpleGrid
+          cols={3}
+          style={{ alignItems: "start", justifyContent: "start" }}
+        >
+          {BannerVariantsStory.render?.(
+            BannerVariantsStory.args as any,
+            context,
+          )}
+        </HvSimpleGrid>
+      </div>
+      {DropdownTestStory.render?.(DropdownTestStory.args as any, context)}
+    </div>
+  ),
+};
+
+export const Test2: StoryObj = {
```

Comment:
maybe we could give these more descriptive names, so that it's clearer in which sample to add a new component/variant

Reply:
Agreed, I'll add some comments

Reply:
I'll add some comments for now